### ISSUE TITLE
Run linter before tests by default

### ISSUE THREAD

**Issue by luqasn** [JLLeitschuh/ktlint-gradle#417]
First of all thanks for this excellent gradle plugin!

We use it in multiple of our projects and have put the following in most of their `build.gradle`s:
```
test.shouldRunAfter ktlintCheck
```
in order to make `./gradlew check` fail as fast as possible. I was wondering if there is a way to set the plugin up in a way to do this by default? My attempt is here: https://github.com/JLLeitschuh/ktlint-gradle/compare/master...luqasn:make-lint-run-before-test?expand=1

I tried to verify the desired behaviour in a test but my knowledge is too limited. But I hope my intentions are clear from this code.

**Comment by Tapchicoma** [comment]
Theoretically `ktlintCheck` (and ktlint tasks that it depends on) should run in parallel with tests (based on available Gradle workers). I am not sure it would be the best approach here to run tests after ktlint checks. Probably your best approach would be to use `./gradlew ktlintCheck test` - this will run them one after another.

@JLLeitschuh what is your opinion on this issue?

**Comment by JLLeitschuh** [comment]
I'd argue that adding a configuration like this would be unnecessary when these require the same amount of code:

`test.shouldRunAfter ktlintCheck`
```
ktlint {
    runBeforeTest = true
}
```

I don't see any value in adding this configuration option, and I don't think this should be the default behaviour.

**Comment by JLLeitschuh** [comment]
Closing this out as a wontfix. Thanks for the feedback though. Always welcome to hear user use cases!
